src/languages/_english.json

```diff
@@ -158,6 +158,7 @@
   "cancel": "Cancel",
   "stability_fee": "Stability fee",
   "liquidation_penalty": "Liquidation fee",
+  "collateral_debt_ceiling": "Collateral debt ceiling",
```

Comment:
I think the context makes it clear enough that this is for collateral, so we can just make it "Debt ceiling"